docs/frontend/developer-guide.md

```diff
@@ -120,7 +120,11 @@ Our apps use chained middlewares to improves code readability, and maintainabili
 
 The Registration1 and Registration code is organized into sub-folders based on a identity provider, an application role, and dashboard folder, or just dashboard folder for routes available for authenticated users without an authorization role. As mentioned, the middleware dynamically rewrites the request URL based on the Auth.js JWT properties of identity_provider and user role so to match our sub-folder structure. So route URL segments such as registration1 `http://localhost:3000/dashboard/operations` would get mapped to nested folder `bciers/apps/registration1/app/(authenticated)/bceidbusiness/industry_user/dashboard/operations' for an authenticated industry user.
 
-For our multi-zone apps, the dashboard app manages the main domain and rewites request to the appropriate zone as required. The dashboard links are configured in `bc_obps/common/fixtures/dashboard/{identity-provider}` and use the `{project/}` pre-fix. So route URL to registration operations would be `http://localhost:3000/registration/dashboard/operations`
+For our multi-zone apps, the dashboard app manages the main domain and rewites request to the appropriate zone as defined in `bciers/apps/dashboard/next.config.js` and `bciers/apps/dashboard/middlewares/withAuthorizationDashboard.ts`. The dashboard links are configured in `bc_obps/common/fixtures/dashboard/{identity-provider}` and use the `{project/}` pre-fix. So route URL to registration operations would be `http://localhost:3000/registration/dashboard/operations`
```

Comment:
I think the example (and maybe other parts of this paragraph?) are outdated. We don't have registration operations anymore, we have `http://localhost:3000/administration/operations`